libs/code-generator/src/lib/__snapshots__/code-generator.spec.ts.snap

```diff
@@ -108,114 +108,32 @@ public class FeatureFilterAttribute : FeatureFilterAttributeBase
 "
 `;
 
-exports[`code-generator > template: dotnet-api > should produce the expected features files > apps/my-app/Project.FeatureBoard.MyApp.csproj 1`] = `""`;
+exports[`code-generator > template: typescript > should produce the expected features files > apps/my-app/featureboard-features.ts 1`] = `
+"export interface SaaSyIconsFeatures {
 
-exports[`code-generator template: dotnet-api should produce the expected features files: apps/my-app/Features.cs 1`] = `
-"using FeatureBoard.DotnetSdk.Attributes;
-using FeatureBoard.DotnetSdk.Models;
-using System.Runtime.Serialization;
+    // Requires attribution
```

Comment:
We should use jsdoc syntax

```
/**
 * Requires attribution
 * @description The span that triggered this job
 */
```

Then not print the `@description` if there is none

Reply:
Done